.github/workflows/build.yml

```diff
@@ -319,12 +371,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        arch: ["386", "amd64", "arm", "arm64"]
+        arch: ["386", "amd64", "arm", "arm64", "s390x"]
       fail-fast: true
     env:
       version: ${{ needs.set-product-version.outputs.product-version }}
       zip_name: consul_${{ needs.set-product-version.outputs.product-version }}_linux_${{ matrix.arch }}.zip
 
+    if: ${{ endsWith(github.repository, '-enterprise') }} || ${{ matrix.arch != 's390x' }}
```

Comment:
so we are no longer verifying at all on OSS?  Is this intentional?

Reply:
Yes, the overhead of running this on OSS is not worth the cost due to the fact the customer requiring this specific build is running ENT only.

Reply:
right sorry, i mis read the `if` and was thinking that no platforms would be verified on OSS.  👍